core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1058,14 +1068,39 @@ object Parser {
 
   /** Repeat 0 or more times with a separator
     */
-  def rep0Sep[A](p1: Parser[A], min: Int, sep: Parser0[Any]): Parser0[List[A]] = {
+  def repSep0[A](p1: Parser[A], min: Int, sep: Parser0[Any]): Parser0[List[A]] = {
```

Comment:
Yes, we should

Reply:
after this is merged, please update the #137 issue when the scalafix is updated so we can safely publish (since I think scala-steward will use that scala fix).

Reply:
The scalafix is updated